src/OpenTelemetry.Extensions.Hosting/OpenTelemetryServicesExtensions.cs

```diff
@@ -193,17 +105,12 @@ public static IServiceCollection AddOpenTelemetrySdk(this IServiceCollection ser
             }
 
             services.AddSingleton(s => createSdk(s));
-            AddOpenTelemetrySdkCore(services);
+            AddOpenTelemetryCore(services);
```

Comment:
`AddOpenTelemetryCore` is a confusing name as it is unclear what does `core` mean. Consider `AddOpenTelemetryInternal`.